src/client/pages/OfferNew/Checkout/UserDetailsForm.tsx

```diff
@@ -105,6 +105,7 @@ export const UserDetailsForm: React.FC<Props> = ({
           name="ssn"
           id="ssn"
           type="number"
+          inputMode="decimal"
```

Comment:
Just curious - why `"decimal"` and not `"numeric"`?

Reply:
Updated now after checking a bit more, numeric and pattern in a combo 🙌